tl-tl/sc/getting-started-java.md

```diff
@@ -2,98 +2,98 @@
-2. Download the Neo Framework Library JAR. The current latest version is here: [org.neo.smartcontract.framework JAR](https://github.com/CityOfZion/neo-java-sdk/blob/master/target/org.neo.smartcontract.framework.jar)
-3. Download an IDE for Java (optional but recommended), e.g. IntelliJ or Eclipse.
-4. Download an IDE for C# - currently the neoj compiler needs to be built manually as it is not in wide distribution release format. Recommended is to get Visual Studio 2017 which is free.
+1. I-download ang Node GUI ng Neo. Sa pagkakasulat nito, nirerekomenda na gumamit ka ng BETA developer GUI dahil mayroon itong mga ekstrang debugging feature na nagpapadali ng lahat. [CoZ NEO GUI](https://github.com/CityOfZion/neo-gui-developer). Ito ay mayroong mga default preset sa Testnet at kinakailangan mong maghintay (hanggang sa ilang oras) para ito ay mag-sync ng lubos.
+2. I-download ang Neo Framework Library JAR. Nandito ang kasalukuyang pinakabago na bersyon: [org.neo.smartcontract.framework JAR](https://github.com/CityOfZion/neo-java-sdk/blob/master/target/org.neo.smartcontract.framework.jar)
+3. Magdownload ng isang IDE para sa Java (opsyonal ito pero rinrekomenda), e.g. IntelliJ o Eclipse.
+4. Magdownload ng isang IDE para sa C# - sa kasalukuyan, ang neoj compiler ay kinakailangang likhain ng mano-mano dahil ito ay hindi naka wide distribution release na format.
 
-## Development Tools
+## Mga Development Tool
 
 ### 1. Visual Studio 2017
 
-If you have already installed Visual Studio 2017 on your computer and checked for .NET Cross-Platform Development at the time of installation, you can skip this section.
+Kong ikaw ay naka-install na ng Visual Studio 2017 sa iyong kompyuter at naka-check sa .NET Cross-Platform Development sa panahon na nag-iinstall, pwede mong laktawan ang seksyon na ito.
 
-Download and install:
+Pag-download at pag-install:
 
-[Visual Studio download address](https://www.visualstudio.com/products/visual-studio-community-vs)
+[Ang address para i-download ang Visual Studio](https://www.visualstudio.com/products/visual-studio-community-vs)
 
-The installation process is very simple, follow the operation prompts step-by-step, it should be noted that you need to check the installation of `.NET Core cross-platform development`, otherwise you will not be able to open neo-vm project in step #3. The installation takes about ten minutes or up to an hour.
+Napakasimple lang ng proseso sa pag-install, sundin lang ang bawat hakbang sa mga operation prompt, dapat mong tandaan na kinakailangan mong e-check ang pag-install ng `.NET Core cross-platform development`, dahil kong hindi, hindi mo maaring buksan ang neo-vm na proyekto na nasa ikatlong hakbang. Ang pag-install ay tumatagal ng halos sampung minuto o hanggang sa isang oras.
```

Comment:
- you could translate `check` to `suriin` or `i-check`
- `dahil kung hindi, hindi mo maaaring`